R/oc_forward.R

```diff
@@ -31,58 +34,89 @@
 #'   (such as "es" for Spanish or "pt-BR" for Brazilian Portuguese). OpenCage
 #'   will attempt to return results in that language. If it is not specified,
 #'   "en" (English) will be assumed by the API.
-#' @param limit The maximum number of results that should be returned. Integer
-#'   values between 1 and 100 are allowed, the default is 10 (\code{oc_forward})
-#'   or 1 (\code{oc_forward_df}), respectively.
-#' @param min_confidence An integer value between 0 and 10 indicating the
-#'   precision of the returned result as defined by its geographical extent,
-#'   (i.e. by the extent of the result's bounding box). See the
-#'   \href{https://opencagedata.com/api#confidence}{API documentation} for
+#' @param limit Numeric vector of integer values to determine the maximum
+#'   number of results returned for each \code{placename}. Integer values
+#'   between 1 and 100 are allowed. Default is 10.
+#' @param min_confidence Numeric vector of integer values between 0 and 10
+#'   indicating the precision of the returned result as defined by its
+#'   geographical extent, (i.e. by the extent of the result's bounding box). See
+#'   the \href{https://opencagedata.com/api#confidence}{API documentation} for
 #'   details. Only results with at least the requested confidence will be
-#'   returned.
-#' @param no_annotations A logical vector indicating whether additional
+#'   returned. Default is \code{NULL}).
+#' @param no_annotations Logical vector indicating whether additional
 #'   information about the result location should be returned. \code{TRUE} by
 #'   default, which means that the output will not contain annotations.
-#' @param no_dedupe A logical vector (default \code{FALSE}), when \code{TRUE}
+#' @param no_dedupe Logical vector (default \code{FALSE}), when \code{TRUE}
 #'   the output will not be deduplicated.
-#' @param no_record A logical vector (default \code{FALSE}), when \code{TRUE} no
-#'   log entry of the query is created and the geocoding request is not cached
+#' @param no_record Logical vector (default \code{FALSE}), when \code{TRUE} no
```

Comment:
`no_record` is not vectorised.
```suggestion
#' @param no_record Logical vector of length one (default \code{FALSE}), when \code{TRUE} no
```